static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic-us-pre-end-of-year.js

```diff
@@ -0,0 +1,207 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic-equal-buttons.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/utils/ajax',
+    'common/modules/commercial/commercial-features',
+    'common/utils/element-inview'
+
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsEpicEqualButtons,
+             robust,
+             arrowRight,
+             config,
+             cookies,
+             ajax,
+             commercialFeatures,
+             ElementInview) {
+
+    return function () {
+        this.id = 'ContributionsEpicUsPreEndOfYear';
+        this.start = '2016-12-06';
+        this.expiry = '2016-12-12';
+        this.author = 'Guy Dawson';
+        this.description = 'Test which Epic variant to use in the US end of year campaign';
+        this.showForSensitive = false;
+        this.audience = 0.1;
+        this.audienceOffset = 0.9;
+        this.successMeasure = 'Conversion rate (contributions / impressions)';
+        this.audienceCriteria = 'All';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'We are able to determine which Epic variant to use in the US end of year campaign';
+        this.canRun = function () {
+            var userHasNeverContributed = !cookies.get('gu.contributions.contrib-timestamp');
+            var worksWellWithPageTemplate = (config.page.contentType === 'Article') && !config.page.isMinuteArticle; // may render badly on other types
+            return userHasNeverContributed && commercialFeatures.canReasonablyAskForMoney && worksWellWithPageTemplate;
+        };
+
+        var contributeUrlPrefix = 'co_global_epic_us_pre_end_of_year';
+        var membershipUrlPrefix = 'gdnwb_copts_mem_epic_us_pre_end_of_year';
+
+        var epicInsertedEvent = 'ContributionsEpicUsPreEndOfYear:insert';
+        var epicViewedEvent = 'ContributionsEpicUsPreEndOfYear:view';
```

Comment:
could we not just have one constant, ContributionsEpicUsPreEndOfYear, that we use throughout rather than the 3 instances of "ContributionsEpicUsPreEndOfYear" that curently exist?